### DISCUSSION TITLE
Allow Battery Upgrade to charge user inventory

### DISCUSSION THREAD

**Discussion by JWTHDYTWA** [P3pp3rF1y/SophisticatedBackpacks#732] [Ideas]
A Battery Upgrade is not that useful as long as it can't charge items even inside the backpack. I suggest:

- making the Battery Upgrade charge the backpack content
- adding the Advanced Battery Upgrade that would charge the main inventory, armor slots and hotbar

**Comment by DJMo13** [comment]
And if possible, it would be awesome if you could add Curious support as well because with the right amount of stack upgrades, it is the biggest portable battery in the game :D

**Comment by Joly0** [comment]
This discussion is already open here https://github.com/P3pp3rF1y/SophisticatedBackpacks/discussions/563
